primitives/rsa/src/main.nr

```diff
@@ -1,26 +0,0 @@
-use dep::std::hash::sha256;
-use dep::biguint::{BigUint56};
-use dep::rsa::{verify_sha256_pkcs1v15};
-
-/*
-+---------+------------------------+--------------+----------------------+
-| Package | Language               | ACIR Opcodes | Backend Circuit Size |
-+---------+------------------------+--------------+----------------------+
-| rsa     | PLONKCSat { width: 3 } | 1642330      | 3020035              |
```

Comment:
Is there a reason why the RSA benchmark should be removed?

If it is not working with v0.26.0, we could lock the `compiler_version` in its Nargo.toml and remove it from the benchmarking scripts, instead of removing the benchmark itself.

Reply:
I was under the impression that Noir does not support old versions. No security vulnerabilities are backported and changes are just applied on top of main without any notion of LTS versions. Therefore it's not a viable solution for a project to stay on some older version. So if someone wants to use the RSA package - they can't as it does not work on the latest Noir and 0.21.0 has security vulnerabilities that are known and only fixed in newer versions. Therefore providing benchmarking results for this version provides a false expectation about a state of things.

Currently - there is no working bignumber implementation that works with the latest version of Noir and therefore - no working RSA implementation.

I'm not complaining, just stating facts and observations. Happy to add it back if it provides some historical context.

Reply:
Good points.

Let's proceed with removing and link to the commit where it existed in the README instead; I'll handle it.

Thank you for sharing your thoughts!